recipe/meta.yaml

```diff
@@ -28,19 +28,24 @@ source:
       - support_new_cuda_layout.diff  # [(linux and cuda_compiler_version == "12.0") or (build_platform != target_platform)]
 
 build:
-  number: 2
+  number: 3
   # TODO: turn on win64 + CUDA 12 once it's ready
   # TODO: turn on aarch64 + CUDA 12 once glibc 2.28 is ready
   skip: true  # [cuda_compiler_version == "None" or (cuda_compiler_version == "12.0" and not linux64)]
   script:
-    - export CUPY_NVCC_GENERATE_CODE="arch=compute_60,code=sm_60;arch=compute_70,code=sm_70;arch=compute_80,code=sm_80"  # [aarch64]
+    - export CUPY_NVCC_GENERATE_CODE="arch=compute_60,code=sm_60;arch=compute_70,code=sm_70;arch=compute_80,code=sm_80"  # [aarch64 or ppc64le]
     # CuPy default detects CUDA from nvcc, but on Conda-Forge's dockers nvcc lives in a different place...
     # With conda-forge/nvcc-feedstock#58, CUDA_PATH is set correctly
     - export NVCC=$(which nvcc)                                                  # [linux]
     {% if cuda_major_minor == (12, 0) %}
```

Comment:
Should this be:
```suggestion
    {% if cuda_major_minor >= (12, 0) %}
```

Reply:
It's OK to add. I'll try to get this PR in, assuming the CIs run fine 🤞 and then apply this change in #208, where John and I are updating the dependencies for linux64 + CUDA 12. Would that be OK?

Reply:
Absolutely. Not a blocker here.

Reply:
Updated in #208.